src/index.html

```diff
@@ -15,18 +15,28 @@
         <h1>Today's To Do</h1>
         <i class="fa-solid fa-arrows-rotate"></i>
       </header>
-      <form action="">
+      <form action="" class="input-form">
         <input
           type="text"
           name="list"
           id="list"
+          class="input-list"
           placeholder="Add to your list..."
         />
         <i class="fa-solid fa-arrow-turn-down fa-rotate-270 fa-sm"></i>
       </form>
       <ul class="list-container">
+        <!-- <li class="list-item">
+          <div class="list-div">
+              <input type="checkbox" class="check" name="" id="">
+              <p>${myToDoLists.description}</p>
+                <input type="text" name="" id="" class="edit--input">
+                <i class="fa-solid fa-ellipsis-vertical" data--option=${myToDoLists.index}></i> -->
+          <!-- </div> -->
+      <!-- </li> -->
```

Comment:
- let's remove this commented code to make our codebase clean and maintainable.

Reply:
Thank you so much for your review @zieeco , I have removed the comments.